DeepSpeech.ipynb

```diff
@@ -616,7 +616,9 @@
     "    batch_x, batch_y, n_steps = batch_set.next_batch()\n",
     "\n",
     "    # Set batch_seq_len for the batch\n",
-    "    batch_seq_len = batch_x.shape[0] * [n_steps]\n",
+    "    multiple = tf.slice(tf.shape(batch_x),[0],[1])\n",
+    "    tile = tf.slice(tf.shape(batch_x),[1],[1])\n",
+    "    batch_seq_len = tf.tile(tile, multiple)\n",
```

Comment:
A tensor doesn't have a member shape and batch_x will eventually be a tensor.

I don't comment on batch_x shape as it's mentioned under the geometric section.